include/mgp.hpp

```diff
@@ -532,7 +532,15 @@ class Map {
   /// operation is undefined.
   void Insert(std::string_view key, Value &&value);
 
-  // void Erase(std::string_view key);  // not implemented (requires mgp_map_erase in the MGP API)
+  /// @brief Updates the `key`-`value` pair in the map. If it doesn't exist inserts it. The `value` is copied.
+  void Update(std::string_view key, const Value &value);
+  /// @brief Updates the `key`-`value` pair in the map. If it doesn't exist inserts it. The `value` is copied.
```

Comment:
Add space between these two

Reply:
Do you mean adding newline in between?
It wasn't added to be consistent with the current code. Ex. Insert few lines above also doesn't have it.

Reply:
just new line after each declaration as we talked on slack